### ISSUE TITLE
vim.utility.preview.markdownPreview.filetypes is adding extra quoting causing the default filetype to not be recognized

### ISSUE THREAD

**Issue by firecat53** [NotAShelf/nvf#496]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked all existing issues and didn't find a similar issue

### Description

I noticed that the markdown-preview seemed to be non-functional with the very simple setting of:

`vim.utility.preview.markdownPreview.enable = true;`

Note the extra single quotes around markdown. This is more easily seen when adding a filetype:
`vim.utility.preview.markdownPreview.filetypes = ["markdown" "vimwiki"];

(See generated init.lua below)
 
I can work around this for now by setting `alwaysAllowPreview = true;`

Thanks!

### 👟 Reproduction steps

Set `vim.utility.preview.markdownPreview.enable = true;`

### 👀 Expected behavior

`:MarkdownPreview` command is available when editing a markdown file.

### 😓 Actual Behavior

`:MarkdownPreview` and associated commands are not available.

### 💻 Metadata

 - system: `"x86_64-linux"`  - host os: `Linux 6.6.63, NixOS, 24.11 (Vicuna), 24.11.20241213.314e12b`  - multi-user?: `yes`  - sandbox: `yes`  - version: `nix-env (Nix) 2.24.10`  - channels(root): `""`  - nixpkgs: `/nix/store/0xbni69flk8380w0apw4h640n37wn1i9-source`

### 📝 Relevant log output

```bash
# Generated init.lua with default settings
vim.g.mkdp_auto_close = true
vim.g.mkdp_auto_start = false
vim.g.mkdp_command_for_global = true
vim.g.mkdp_filetypes = {"'markdown'"}
vim.g.mkdp_open_ip = ""
vim.g.mkdp_open_to_the_world = false
vim.g.mkdp_port = ""
vim.g.mkdp_refresh_slow = false

# Generated init.lua (just the filetypes setting) with additional filetypes set:
vim.g.mkdp_filetypes = {"'markdown', 'vimwiki'"}
```

**Comment by NotAShelf** [comment]
Could you see if #498 fixes the issue?

**Comment by firecat53** [comment]
Fixed by #498, thanks!
